mlflow/pyfunc/model.py

```diff
@@ -332,7 +353,7 @@ def _save_model_with_class_artifacts_params(
     _PythonEnv.current().to_yaml(os.path.join(path, _PYTHON_ENV_FILE_NAME))
 
 
-def _load_pyfunc(model_path):
+def _load_pyfunc(model_path: str, model_config: Dict[str, Any] = None):
```

Comment:
Same here, we should use **kwargs here to support extra configs passed in during load_model.
We should read load_config from the model file first, then update with additional passed-in configs.

Reply:
Same comments as above, kwargs is a dangerous implementation for future compatibility. It was discussed in the [original doc](https://docs.google.com/document/d/1sLR6SCby0hLRLSHonPbnzLpS-7_J4Clk/edit?usp=sharing&ouid=106956464969714561112&rtpof=true&sd=true) we shared.